wine/tricks.go

```diff
@@ -2,10 +2,34 @@ package wine
 
 import (
 	"log"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/vinegarhq/vinegar/internal/config"
 )
 
+func isWinerootTricksAvaliable(wineroot string) (bool, error) {
+	_, err := exec.LookPath(filepath.Join(wineroot, "bin", "winetricks"))
```

Comment:
Unnecessary, the existing `Run()` will already tell us that, and you're not wrapping the error so there is no point.